docs/packages.md

```diff
@@ -37,6 +37,7 @@ A package source consists of a directory containing at least two files:
 - `image` _(string)_: *(mandatory)* The name of the image to build
 - `org` _(string)_: The hub/registry organisation to which this package belongs
 - `arches` _(list of string)_: The architectures which this package should be built for (valid entries are `GOARCH` names)
+- `sources` _(list of strings)_: Optionally specifies sources for the package build which are outside the package directory. The format is `src:dst`, where `src` can be a relative path to the package directory and `dst` is the destination in the build context. This is useful for sharing files, such as vendored go code, between packages.
```

Comment:
I think `extra-sources` would be a more accurate name.

> where `src` can be a relative path to the package directory

"can" or "must"? I would also word it as "path relative to the".